packages/protocol/contracts/team/airdrop/ERC721Airdrop.sol

```diff
@@ -40,7 +40,15 @@ contract ERC721Airdrop is MerkleClaimable {
         vault = _vault;
     }
 
-    function _claimWithData(bytes calldata data) internal override {
+    function _claimWithData(
```

Comment:
ERC721Airdrop shall not be changed at all -- delegation shall not be something MerkleClaimable shall care.

Reply:
Then how do you achieve the desired one transaction per airdrop ? I already told you and Keng i prefer the 2 time transactions, because of errors too for example (UI does not consturcts sig things properly, etc.)